custom_dc/load_data.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+# Script to process stats using simple stats loader.
+# Defaults
+MODE="customdc"
+OUTPUT_DIR=".data/output_$MODE"
+USAGE="Script to load data into custom dc using simple importer.
+Usage: $(basename $0) [Options]
+Options:
+  -c <file>       Json config file for stats importer
+  -e <file>       Load environment variables from file.
+  -i <dir>        Input directory to process
+  -k <api-key>    DataCommons API Key
+  -o <dir>        Output folder for stats importer. Default: $OUTPUT_DIR
+  -m <customdc|maindc> Mode of operation for simple importer. Default: $MODE
+  -j <jar>        DC Import java jar file.
+                    Download latest from https://github.com/datacommonsorg/import/releases/
+  -r <steps>      Steps to run. Can be one or more from:
+                    stats, validate
+  -s <cloud_sql>  Cloud SQL instance. Also set DB_USER and DB_PASS with -u, -p
+  -u <username>   DB username for cloud SQL. Default: $DB_USER.
+  -p <password>   DB password for cloud SQL. Default: $DB_PASS.
+
+For more, please refer to https://github.com/datacommonsorg/import/tree/master/simple
```

Comment:
Let's publish a doc for this in docsite and point to that instead. I can give you a link later.